README.txt

```diff
@@ -68,6 +68,16 @@ Easy Way
 
     $ pip install gssapi
 
+You can also use a specific Kerberos library by defining the `GSSAPI_*`
+environment variables:
+
+.. code-block:: bash
+
+    $ export PATH=/path/to/where/krb5-config/is:$PATH
+    $ export GSSAPI_LINKER_ARGS="$(krb5-config --libs gssapi)"
+    $ export GSSAPI_COMPILER_ARGS="$(krb5-config --cflags gssapi)"
+    $ export GSSAPI_MAIN_LIB="/path/to/where/libgss{{api,-3}.dll,api.{so,dylib}}/is"
```

Comment:
> I can't say I use this with Windows so I can't really speak more, I just think it should be more obvious that the default is libgssapi.so for Linux and libgssapi.dylib for macOS as per 

I really don't think this is clearer @jborean93

Reply:
My main comment was to do document `libgssapi.so` and just add a comment saying this could be `.dylib` or something else on other platforms.